configure.py

```diff
@@ -1654,11 +1655,8 @@ def make_bin_override():
 configure_library('zlib', output)
 configure_library('http_parser', output)
 configure_library('libuv', output)
-configure_library('libcares', output)
-configure_library('nghttp2', output)
-# stay backwards compatible with shared cares builds
-output['variables']['node_shared_cares'] = \
-    output['variables'].pop('node_shared_libcares')
+configure_library('cares', output, pkgname='libcares')
+configure_library('nghttp2', output, pkgname='libhttp2')
```

Comment:
```suggestion
configure_library('nghttp2', output, pkgname='libnghttp2')
```

Is this a typo?

Reply:
Yes, it is. Can't believe that slipped through.